package.json

```diff
@@ -12,7 +12,9 @@
     "unit": "mocha test/",
     "test": "npm run format && npm run unit && npm run cover && npm run check-coverage"
   },
-  "author": "Evan Shortiss",
+  "author": {
+    "name": "Evan Shortiss"
+  },
```

Comment:
Can you add a new field here "contributors" and add yourself? Here's the docs: https://docs.npmjs.com/files/package.json#people-fields-author-contributors